src/ntrip_client/ntrip_client.py

```diff
@@ -145,26 +151,34 @@ def connect(self):
 
     # Some debugging hints about the kind of error we received
     known_error = False
+    error_code = self.ERROR_CODE_NO_ERROR
```

Comment:
```suggestion
    error_flags = self.ERROR_FLAG_NO_ERROR
```
Direct assignment here is fine since you're clearing all flags.